<launch>
  <node pkg="romina2" type="pilot" name="pilot">
    <remap to="/cmd_vel_mux/input/teleop" from="/cmd_vel"/>
  </node>
  <node pkg="tf" type="static_transform_publisher" name="cam_broadcaster" args=".063 0 0.15 -1.57 0 -1.57 robot usb_cam 100" />
 
  <node pkg="nodelet" type="nodelet" name="manager" args="manager"/>
	<node pkg="nodelet" type="nodelet" name="cam" args="load usb_cam/usb_cam_nodelet manager">
	    <param name="video_device" value="/dev/video0" />
	    <param name="camera_info_url" type="string" value="file://$(find romina2)/cfg/head_camera.yaml"/>
	    <param name="image_width" value="160" />
	    <param name="image_height" value="120" />
	    <param name="camera_frame_id" value="usb_cam" /> 
	    <param name="framerate" value="2" />
	    <param name="pixel_format" value="yuyv" />
	    <param name="io_method" value="mmap" />
  	</node>
	<node pkg="nodelet" type="nodelet" name="ar_pose" args="load ar_pose/ARMultiNodelet manager">
		<param name="marker_pattern_list" type="string" value="$(find testarpose)/patterns/column_markers"/>
		<param name="threshold" type="int" value="125"/>
		<param name="publish_tf" type="bool" value="true"/> 
		<remap to="/cam/camera_info" from="/usb_cam/camera_info" />
		<remap to="/cam/image_raw" from="/usb_cam/image_raw" />
	</node>
</launch>
